cryk-ui/src/components/CustomNavbar.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import { Container, Navbar } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
 import "./css/CustomNavbar.css";
 
 export default function CustomNavbar() {
+  const navigate = useNavigate();
+
   return (
     <Navbar collapseOnSelect expand="lg" className="navbar">
       <Container>
-        <Navbar.Brand className="me-auto" href="/">CRYK</Navbar.Brand>
+        <h2 className="navbar-brand" onClick={() => { navigate(`/`) }}>CRYK</h2>
```

Comment:
We have to use `navigate` from `react-router-dom` in order to navigate pages. Otherwise, the page will refresh and we'll use the global context (we use it for authentication)